diff --git a/dist/runtime/components/Prose/ProseA.vue b/dist/runtime/components/Prose/ProseA.vue
index 73f8bcaf4c1f4ce974b4384b46b4edb5a7bf6286..a9bfdb7f1e7612feaa6ddbfd6a2f4fd394f79aa1 100644
--- a/dist/runtime/components/Prose/ProseA.vue
+++ b/dist/runtime/components/Prose/ProseA.vue
@@ -13,7 +13,7 @@ defineProps({
 </script>
 
 <template>
-  <NuxtLink :href="href" :target="target">
+  <NuxtLink :to="href" :target="target">
     <slot />
   </NuxtLink>
 </template>
